android/src/main/java/com/terveystalo/react_native/matomo_sdk/RNMatomoSdkModule.java

```diff
@@ -19,4 +32,57 @@ public RNMatomoSdkModule(ReactApplicationContext reactContext) {
   public String getName() {
     return "RNMatomoSdk";
   }
+
+  @ReactMethod
+  @SuppressWarnings("unused")
+  public void initialize(String apiUrl, int siteId, Promise promise) {
+    try {
+      if (tracker == null) {
+        tracker = TrackerBuilder
+                .createDefault(apiUrl, siteId)
+                .build(Matomo.getInstance(this.reactContext));
+
+        // Disable default automatic user ID generation for consistency with iOS
+        tracker.setUserId(null);
+      }
+      promise.resolve(null);
+    } catch(Exception e) {
```

Comment:
Do we want to actually catch all errors?

Reply:
Yeah, I think it is better to propagate all errors to the JS runtime, and handle them there instead.